api/docker/usr/local/share/cron/run-every-x-seconds.sh

```diff
@@ -7,18 +7,19 @@ error="no"
 
 if [ 'x'"$runCommand" != 'x' ]
 then
-    if [ 'x'$inputPeriod != 'x' ]
+    if [ 'x'"$inputPeriod" != 'x' ]
     then
-        loops=$(( $RUN_TIME / $inputPeriod ))
+        loops=$(( RUN_TIME / inputPeriod ))
```

Comment:
Really, we don't need the `$`s here?!

Reply:
Seems to work both ways:
```bash
bash-3.2$ export RUN_TIME=10
bash-3.2$ export inputPeriod=2
bash-3.2$ loops=$(( RUN_TIME / inputPeriod ))
bash-3.2$ echo $loops
5
bash-3.2$ loops=$(( $RUN_TIME / $inputPeriod ))
bash-3.2$ echo $loops
5
```
https://github.com/koalaman/shellcheck/wiki/SC2004 has the explanation why they recommend this change